components/footer.jsx

```diff
@@ -9,7 +9,7 @@ var Footer = React.createClass({
         <div className="footer">
           <div className="row">
             <div className="half">
-              <img src="/images/Mozilla-wordmark.png"/>
+              <img src="/images/mozilla.svg"/>
```

Comment:
Updated to new image file provided by Jordan.

It's and svg, smaller, and is a bit lighter in colour.